osquery/sql/sqlite_string.cpp

```diff
@@ -210,6 +210,73 @@ static void regexStringMatchFunc(sqlite3_context* context,
                       SQLITE_TRANSIENT);
 }
 
+static void concatFunc(sqlite3_context* context,
+                       std::string sep,
+                       int starting,
+                       int argc,
+                       sqlite3_value** argv) {
+  // Nothing to concat, early return
+  if ((argc - starting) == 0) {
+    return;
+  }
+
+  std::string output;
+
+  for (auto i = starting; i < argc; i++) {
+    if (SQLITE_NULL == sqlite3_value_type(argv[i])) {
+      continue;
+    }
+
+    output.append(reinterpret_cast<const char*>(sqlite3_value_text(argv[i])));
+
+    if (sep != "" && i + 1 < argc) {
+      output.append(sep);
+    }
+  }
+
+  // Give up if the output is so large it's length overflows int
+  if (output.size() > std::numeric_limits<int>::max()) {
+    LOG(INFO) << "Too much data for concat_ws";
```

Comment:
I don't think we need this `INFO` here since the error is set in the result code.

Reply:
I've removed the double reporting. I did not update the existing functions here.